### PR TITLE
Updated French translation

### PR DESCRIPTION
**Pull request by SolarLiner**

### CODE REVIEW

**Comment by charlesg99** [comment]
Hey @SolarLiner! I spotted a small typo. At line 216, I think it should be "Les titres d'application ont _tous_". Thanks

**Comment by SolarLiner** [comment]
Good catch @charlesg99, I fixed the typo.

File: po/fr.po [hunk 1]
```diff
@@ -131,319 +144,451 @@ msgid "Highlight focused application"
...
+msgid "Indicator height (px)"
+msgstr "Hauteur indicateur (px)"
 
 #: Settings.ui.h:20
+msgid "0"
+msgstr "0"
+
+#: Settings.ui.h:21
+msgid "Indicator color - Override Theme"
+msgstr "Couleurde l'indicateur - Remplacer le thème"
```
Comment (franglais125) [inline po/fr.po]: One space missing here :)

File: po/fr.po [hunk 2]
```diff
@@ -131,319 +144,451 @@ msgid "Highlight focused application"
...
+msgid "Font size (px) of the application titles (default is 14)"
+msgstr "Taille (px) du texte des titres d'application (défaut: 14)"
+
+#: Settings.ui.h:29
+msgid "Font color of the application titles"
+msgstr "Couleur du texte des titres d'application"
+
+#: Settings.ui.h:30
+msgid "Maximum width (px) of the application titles (default is 160)"
+msgstr "Longueur maximum (px) des titres d'applications (défaut: 160)"
```
Comment (franglais125) [inline po/fr.po]: `titres d'applications` should perhaps just be
`titres d'application` as in the other translations above ?

File: po/fr.po [hunk 3]
```diff
@@ -131,319 +144,451 @@ msgid "Highlight focused application"
...
+
+#: Settings.ui.h:31
+msgid "Use a fixed width for the application titles"
+msgstr "Utiliser une longueur fixe pour les titres d'application"
```
Comment (LaurentTreguier) [inline po/fr.po]: Should `width` be translated as `longueur` or `largeur` ?

File: po/fr.po [hunk 4]
```diff
@@ -131,319 +144,451 @@ msgid "Highlight focused application"
...
+#: Settings.ui.h:42
+msgid "Window peeking mode opacity"
+msgstr "Opacité de la vue de fenêtre"
+
+#: Settings.ui.h:43
+msgid ""
+"All windows except for the peeked one have their opacity set to the same "
+"value."
+msgstr ""
+"Toute fenêtre, mis à part celle prévisualisée, ont leur opacité mise à la "
```
Comment (LaurentTreguier) [inline po/fr.po]: `Toute fenêtre` should maybe be `Toutes les fenêtres` if `ont` is used afterwards

File: po/fr.po (hunk 3 above)
Comment (SolarLiner) [reply po/fr.po]: `longueur` is consistent with my other translation. I don't think `largeur` would be so much more accurate as to part from the consistency.

File: po/fr.po [hunk 5]
```diff
@@ -276,8 +277,8 @@ msgid ""
 "All windows except for the peeked one have their opacity set to the same "
 "value."
 msgstr ""
-"Toute fenêtre, mis à part celle prévisualisée, ont leur opacité mise à la "
-"même valeur."
+"Toute les fenêtres, mis à part celle prévisualisée, ont leur opacité mise à "
```
Comment (franglais125) [inline po/fr.po]: Thanks for all the updates!

Still, a small missing `s` over here :). `Toutes`.

Merci!

File: po/fr.po [hunk 6]
```diff
@@ -104,7 +117,7 @@ msgstr "Cycler sur les fenêtres + réduire"
 
 #: Settings.ui.h:9
 msgid "Behavior for Middle-Click."
-msgstr "Comportement du clic molette"
+msgstr "Comportement du clic molette."
```
Comment (AdrienPoupa) [inline po/fr.po]: Why did you add a dot there?

File: po/fr.po [hunk 7]
```diff
@@ -131,319 +144,452 @@ msgid "Highlight focused application"
...
+"l'aperçu peut rester trop longtemps quand on passe à une icône adjacente."
 
-#: Settings.ui.h:27
+#: Settings.ui.h:37
+msgid "Time (ms) before showing (100 is default)"
+msgstr "Temps (ms) avant d'afficher (100 par défaut)"
+
+#: Settings.ui.h:38
+msgid "Enable window peeking"
+msgstr "Activer la vue de fenêtre"
```
Comment (AdrienPoupa) [inline po/fr.po]: Sounds weird, I'd use something like "Activer la prévisualisation" or "miniature"

File: po/fr.po (hunk 6 above)
Comment (SolarLiner) [reply po/fr.po]: It is good practice to follow the original message's structure. There's a dot in the original message, therefore there is a dot in the translated message.

Furthermore, POEdit complains if I don't put it there...

Comment (AdrienPoupa) [reply po/fr.po]: Sure :)

File: po/fr.po (hunk 7 above)
Comment (SolarLiner) [reply po/fr.po]: "Window peeking" is definitely weird to translate though. I did not remember how it was called in Windows (given the obvious inspiration). "prévisualisation" seems a good word, and I have used it elsewhere too.

**Comment by AdrienPoupa** [comment]
Thanks for making this extension even better!
